### PR TITLE
Graphene spec

### PR DESCRIPTION
**Pull request by gbiss**

### CODE REVIEW

File: 091.mediawiki [hunk 1]
```diff
@@ -1,33 +1,317 @@
+
```
Comment (sickpig) [inline 091.mediawiki]: You should remove the changes you made to BUIP 091 from this pull

File: 091.mediawiki [hunk 2]
```diff
@@ -1,33 +1,317 @@
...
+The vAdditionalTxs field should include at least the coinbase transaction, since it is not possible for it to be in the receiver's mempool.
+
+====CGrapheneSet====
+''CGrapheneSet'' contains all information critical to the graphene set reconciliation process, absent any block-specific details. It is the only non-standard data structure in ''CGrapheneBlock''.
+
+{|
+ |'''Field Name'''||'''Type'''||'''Size'''||'''Purpose'''
+ |-
+ |ordered||bool||1 byte||True if order is important in the set
```
Comment (awemany) [inline 091.mediawiki]: Just for flexibility as noted in the chat, I would change this from type `bool` to type `byte` and say that currently, only the value of 0 has a well-defined meaning.

Comment (gbiss) [reply 091.mediawiki]: Done.

**Comment by gbiss** [comment]
Deprecating in favor of new PR:
https://github.com/BitcoinUnlimited/BUIP/pull/83

**Pull request closed without merging**